src/components/organisms/BannerAdmin/BannerAdmin.scss

```diff
@@ -0,0 +1,53 @@
+@import "scss/constants.scss";
+
+.BannerAdmin {
+  &__input-text {
+    text-align: start;
+    padding: $spacing--sm $spacing--lg;
+    font-size: $font-size--md;
+
+    &--large {
+      height: 55px;
+    }
+  }
+
+  &__input-container {
+    margin-bottom: $spacing--lg;
+  }
+
+  &__button-container {
+    display: flex;
+    justify-content: space-between;
+    margin-top: $spacing--xl;
+  }
+
+  &__button {
+    width: $button-width;
+  }
+
+  &__action-container {
+    margin-left: 50px;
+  }
+
+  &__checkbox {
+    justify-content: start;
+    margin-bottom: $spacing--lg;
+
+    &--action {
+      margin-bottom: $spacing--xs;
+    }
+  }
+
+  &__checkbox-label {
+    font-size: $font-size--lg;
+  }
+}
+
+// Override global styles button
+button.BannerAdmin__button {
+  background-color: $primary--live;
+
+  &--close {
+    background-color: $red;
+  }
+}
```

Comment:
Hrmm.. not sure about this.. I think @denisdimitrov @sunny-viktoryia @mike-lvov or similar may have used a better pattern to increase specificity here on other PR's maybe?

Reply:
you can find the example of how I handled that here: 
https://github.com/sparkletown/sparkle/blob/0b519dddd0e5bcf3f3afd7107c6e4c55e72048c5/src/components/organisms/ProfileModal/EditProfileLinkForm/EditProfileLinkForm.scss#L37:L46

Reply:
fixed 1c2d835